doc/news/DM-46458.feature.rst

```diff
@@ -0,0 +1,3 @@
+* In ``atcalsys_schema.yaml``, add default values for wavelength, entrace_slit, exit_slit. Add option to set monochromator_grating to None to skip monchromator configuration and set None as default value."
```

Comment:
- Remove the extra quote at the end of this item.
- "... entrace_slit, exit_slit." > "... entrace_slit and exit_slit."
- Consider using the semantic breaks.